src/py/flwr/server/app.py

```diff
@@ -568,9 +571,17 @@ def _run_fleet_api_grpc_rere(
     state_factory: StateFactory,
     certificates: Optional[Tuple[bytes, bytes, bytes]],
     interceptors: Optional[Sequence[grpc.ServerInterceptor]] = None,
+    num_workers: int = 1,
 ) -> grpc.Server:
     """Run Fleet API (gRPC, request-response)."""
     # Create Fleet API gRPC server
+    if num_workers != 1:
+        raise ValueError(
+            f"The supported number of workers for the Fleet API is "
+            f"1. Instead given {num_workers}. The functionality of >1 workers will be "
+            f"added in the future releases."
+        )
+
```

Comment:
I wonder if we should do the check if `num_workers != 1` earlier when parsing the argparse (in line 315 for instance?) . To me it feels more natural to raise the error there as opposed to inside this function (same applies to `_run_fleet_api_rest`).

Reply:
Moved to `run_superlink`.